README.rst

```diff
@@ -240,6 +240,31 @@ Then run with:
 
 ``$ sh savify.sh "https://open.spotify.com/track/4Dju9g4NCz0LDxwcjonSvI"``
 
+You also have the option to quickstart using our docker script which have an integrated VPN-check
+to see if there are any VPN containers the script can connect to. The script is perfect for being
+scheduled with cron:
+
+.. code-block:: sh
+
+    $ wget https://github.com/laurencerawlings/savify/savify-docker-scripts.sh
```

Comment:
https://github.com/laurencerawlings/savify//latest/download/savify-docker-scripts.zip

This would be the format of the link

Reply:
Perfect, thank you!